### DISCUSSION TITLE
Πρόγραμμα συναντήσεων Παρ. 22 Οκτ και ώρα συμμετοχής για κάθε ΑΜ

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/hci#1584] [Announcements]
Στις σημερινές συναντήσεις θα γίνει συζήτηση σε ομάδες όπως περιγράφεται στο αντίστοιχο κριτήριο:
https://courses-ionio.github.io/projects/classroom/

Το θέμα των ομαδικών συζητήσεων είναι η τρέχουσα ύλη και οι απαντήσεις στα ασύγχρονα κουϊζ της προηγούμενης εβομάδας

Η συμμετοχή στις συναντήσεις θα γίνει σε μικρές ομάδες 10-15 ατόμων σύμφωνα με το ΑΜ σας και το παρακάτω ωριαίο πρόγραμμα, το οποίο προέκυψε από τις δηλώσεις προτίμησης https://github.com/courses-ionio/hci/discussions/1578

## 3-4μμ
Π2020023
Π2019215
Π2020086
Π2020154
Π2020163
Π2020054
Π2020071
Π2020165 
Π2020094
Π2020210
Π2020085
π2019009
Π2020012
Π2020072
Π2018015
Π2018140
Π2018060
Π2020204

## 4-5μμ
Π2020231
Π2020060
Π2020149
Π2019135
Π2020106
Π2019015
Π2020011
Π2018012
P2020155
P2019213
Π2020148
 Π2020102
Π2020194
Π2020038
Π2020124
Π2018073
Π2018013
Π2020143
Π2019137
Π2020019

## 5-6μμ
Π2020126
Π2020171
Π2020114
Π2020233 
Π2020160
π2020161
π2020111
Π2020205
Π2020078
Π2020132
Π2020030
Π2020017
Π2019032
Π2019170
Π2019153
Π2019083
Π2015060

## 6-7μμ
Π2018143
Π2020025
P2008999
Π2020068
P2020062
p2016046
Π2014148
Π2020158
Π2020150
Π2019214
Π2020156
Π2019165
π2020061
Π2017089
P2018209

## 7-8μμ
Π2016072
P2016152
p2020134
Π2020174
Π2019152
Π2020152
Π2020197
Π2020130
Π2020064
Π2018040
Π2020008
Π2020051
Π2020044
Π2020070
π2020173
Π2020104
Π2019097
Π2018069

**Comment by giannisbarlas1** [comment]
κυριε το ΑΜ μου δεν φαινεται σε καποιες απο τις παραπανω ωρες παρολου που συμπληρωσα τις δηλωσεις προτιμησης

**Reply by epidrome** [reply]
δεν υπάρχει πρόβλημα!

αρκετοί δήλωσαν μετά την δημιουργία της ανακοίνωσης 2μμ σήμερα, οπότε θα προστεθούν μέχρι αύριο το πρωί στην παραπάνω λίστα

δεχόμαστε και αδήλωτους (της τελευταίας στιγμής!) αρκεί να μην είναι υπερβολικά πολλοί σε μια ώρα και χαλάσει η οργάνωση

αφού έτσι και αλλιώς θα πρέπει να πάρουμε παρουσίες με την άφιξη στην αίθουσα

**Reply by p14kairas** [reply]
Για όσους δεν βρίσκονται Κέρκυρα τι γίνεται?

**Reply by epidrome** [reply]
η παρουσία στα μαθήματα δεν είναι υποχρεωτική, φυσικά βοηθάει στην κατανόηση και στην τελική εξέταση

**Comment by epidrome** [comment]
η ανακοίνωση έχει ενημερωθεί με όσους έκαναν δήλωση μέχρι τώρα

**Reply by NamesAreReallyHard** [reply]
Το δικό μου δεν το έρχεται βάλει ούτε στο ενημερωμένο Π2020090
